Tests/SwiftDocCTests/Model/SemaToRenderNodeTests.swift

```diff
@@ -2305,29 +2305,29 @@ Document @1:1-11:19
     }
 
     let asidesStressTest: [RenderBlockContent] = [
-        .aside(style: .note, content: [.paragraph(inlineContent: [.text("This is a note.")])]),
-        .aside(style: .tip, content: [.paragraph(inlineContent: [.text("Here’s a tip.")])]),
-        .aside(style: .important, content: [.paragraph(inlineContent: [.text("Keep this in mind.")])]),
-        .aside(style: .experiment, content: [.paragraph(inlineContent: [.text("Try this out.")])]),
-        .aside(style: .warning, content: [.paragraph(inlineContent: [.text("Watch out for this.")])]),
-        .aside(style: .attention, content: [.paragraph(inlineContent: [.text("Head’s up!")])]),
-        .aside(style: .author, content: [.paragraph(inlineContent: [.text("I wrote this.")])]),
-        .aside(style: .authors, content: [.paragraph(inlineContent: [.text("We wrote this.")])]),
-        .aside(style: .bug, content: [.paragraph(inlineContent: [.text("This is wrong.")])]),
-        .aside(style: .complexity, content: [.paragraph(inlineContent: [.text("This takes time.")])]),
-        .aside(style: .copyright, content: [.paragraph(inlineContent: [.text("2021 Apple Inc.")])]),
-        .aside(style: .date, content: [.paragraph(inlineContent: [.text("1 January 1970")])]),
-        .aside(style: .invariant, content: [.paragraph(inlineContent: [.text("This shouldn’t change.")])]),
-        .aside(style: .mutatingVariant, content: [.paragraph(inlineContent: [.text("This will change.")])]),
-        .aside(style: .nonMutatingVariant, content: [.paragraph(inlineContent: [.text("This changes, but not in the data.")])]),
-        .aside(style: .postcondition, content: [.paragraph(inlineContent: [.text("After calling, this should be true.")])]),
-        .aside(style: .precondition, content: [.paragraph(inlineContent: [.text("Before calling, this should be true.")])]),
-        .aside(style: .remark, content: [.paragraph(inlineContent: [.text("Something you should know.")])]),
-        .aside(style: .requires, content: [.paragraph(inlineContent: [.text("This needs something.")])]),
-        .aside(style: .since, content: [.paragraph(inlineContent: [.text("The beginning of time.")])]),
-        .aside(style: .todo, content: [.paragraph(inlineContent: [.text("This needs work.")])]),
-        .aside(style: .version, content: [.paragraph(inlineContent: [.text("3.1.4")])]),
-        .aside(style: .throws, content: [.paragraph(inlineContent: [.text("A serious error.")])]),
+        .aside(style: .init(displayName: "Note"), content: [.paragraph(inlineContent: [.text("This is a note.")])]),
+        .aside(style: .init(displayName: "Tip"), content: [.paragraph(inlineContent: [.text("Here’s a tip.")])]),
+        .aside(style: .init(displayName: "Important"), content: [.paragraph(inlineContent: [.text("Keep this in mind.")])]),
```

Comment:
Have we verified/unit tested that the existing behavior regarding capitalization hasn't changed?

For example, I'd expect that:

```markdown
> important: Text
```

still renders as an aside of style "important" with a capitalized "Important" title.

Also, that

```
> invariant: Text
```

Still renders an aside with a capitalized "Invariant", since was the previous behavior.